src/MapSettings.hpp

```diff
@@ -161,6 +161,9 @@ struct MapSettings {
    */
   bool fade_traffic;
 
+  /** Show FLARM according to climb rate and relative altitude */
+  bool use_detailed_flarm_colours;
```

Comment:
This commit adds a new option but doesn't do anything with it. The commit is incomplete.